subprojects/android-test/test-screenshot/src/main/java/com/avito/android/test/screenshot_test/internal/ScreenshotDirectories.kt

```diff
@@ -0,0 +1,44 @@
+package com.avito.android.test.screenshot_test.internal
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.os.Bundle
+import com.avito.android.screenshot_test.internal.ScreenshotRunner.SDCARD_DIRECTORY
+import java.io.File
+
+class ScreenshotDirectories(private val context: Context) {
```

Comment:
Context makes this class harder to test.
Can we pass only what we need?

Reply:
fixed here